packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -16760,6 +16760,100 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+    delete:
+      tags:
+        - purposeTemplates
+      summary: Delete the Document of a Risk Analysis Template Form Answer Annotation
+      description: Delete the Document of a Risk Analysis Template Form Answer Annotation
```

Minor, align with description of get call above (or align that one to this)
```suggestion
      summary: Delete the Document of a Risk Analysis Form Template Answer Annotation
      description: Delete the Document of a Risk Analysis Form Template Answer Annotation
```

---

👍 Done here: [da31e98](https://github.com/pagopa/interop-be-monorepo/pull/2581/commits/da31e9845c1bab9d85e75e1730f6bb76aa519d67)